march_gait_selection/src/march_gait_selection/dynamic_gaits/balance_gait.py

```diff
@@ -88,5 +89,7 @@ def __getitem__(self, name):
             return self.calculate_trajectory('right_leg')
         elif name == 'left_swing':
             return self.calculate_trajectory('left_leg')
+        elif name in self.default_walk:
+            return self.default_walk[name]
         else:
             return None
```

Comment:
You can simply do
```suggestion
        else:
            return self.default_walk[name]
```
(self.default_walk[name] returns None if name not in self.default_walk)

Reply:
simple yet elegant, good solution.